### PR TITLE
Add BEC team codeowner

### PR DESCRIPTION
**Pull request by qnkhuat**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -8,3 +8,5 @@ enterprise/backend/src/metabase_enterprise/task/truncate_audit_table.clj @noahmo
 enterprise/backend/src/metabase_enterprise/internal_user.clj @noahmoss
 src/metabase/**/*permissions* @noahmoss
 src/metabase/integrations @noahmoss
+src/sync @qnkhuat @calherries
```
Comment (qnkhuat) [inline .github/CODEOWNERS:11]: @calherries took the liberty to add you to this as well

**Comment by piranha** [comment]
@qnkhuat do you think it makes sense to add more of us around in one go? Like me for serialization etc

**Comment by qnkhuat** [comment]
yea, added you as owner for serialization. I tried to add one for caching, but not sure what's the code path is, haha. maybe after we unify the API :).

@calherries , @crisptrutski for upload.

**Review by tsmacdonald — APPROVED** [review]
LGTM

Added the `backport` label since it may as well trigger on backports, too

**Pull request merged**